test/clj/rems/auth/test_oidc.clj

```diff
@@ -0,0 +1,116 @@
+(ns ^:integration rems.auth.test-oidc
+  (:require [clj-http.client]
+            [clojure.test :refer [deftest is testing use-fixtures]]
+            [rems.api.testing :refer [api-fixture]]
+            [rems.auth.oidc :as oidc]
+            [rems.config]
+            [rems.ga4gh]
+            [rems.jwt]
+            [rems.json :as json]
+            [rems.testing-util :refer [with-fake-login-users]]))
+
+(use-fixtures :once api-fixture)
+
+(deftest test-user-does-not-exist
```

Comment:
these two tests `test-user-does-not-exist` and `test-user-has-no-name` seem to be identical now?

Reply:
There is a lot of setup involved to run them which can be perhaps shared. I'll see what I can do.